README.md

```diff
@@ -247,6 +248,15 @@ proc set_ping_interval*(ctx: MqttCtx, txInterval: int) =
 
 Set the clients ping interval in seconds. Default is 60 seconds.
 
+____
+
+### set_ssl_certificates*
+
+```nim
+proc set_ping_interval*(ctx: MqttCtx, sslCertFile: string, sslKeyFile: string) =
```

Comment:
Proc name should be `set_ssl_certificates` instead of  ~set_ping_interval~

Reply:
Fixed :+1: